src/scrapers/platformen/mijnbuurtje.py

```diff
@@ -0,0 +1,180 @@
+import json
+import re
+from datetime import date
+from typing import Generator
+
+from lxml import etree
```

Comment:
Kijk even naar defusedxml voor wat veiligere xml handling in Python https://pypi.org/project/defusedxml/#python-xml-libraries

Reply:
Zo te zien een vrij eenvoudige drop in replacement ook.

Reply:
Niet zo eenvoudig dus. Het lijkt er op dat ik nu `etree.HTML(self.html)` moet vervangen voor iets als `ElementTree.fromstring(self.html)`. Dit klapt er bij de eerste de beste poging meteen uit met de melding `xml.parsers.expat.ExpatError: unbound prefix: line 3, column 263`

Dit is partial HTML wat ik hier probeer te parsen. En de HTML() is denk een stuk vergeeflijker. Ik heb gezocht naar instellingen om dat misschien aan te passen. Maar ik kan alleen maar wat security opties aan en uitzetten voor zover ik kan zien.

Reply:
Issue voor gemaakt nu vervanging niet een eenvoudige drop-in blijkt te zijn: #98